Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -41,6 +45,8 @@ public final class GestureManager: GestureHandlerDelegate {
             gestureOptions.quickZoomEnabled = quickZoomGestureRecognizer.isEnabled
             gestureOptions.panMode = panGestureHandler.panMode
             gestureOptions.panDecelerationFactor = panGestureHandler.decelerationFactor
+#warning("TODO: which of many handlers should be source of truth here")
+            gestureOptions.focalPoint = doubleTapToZoomInGestureHandler.focalPoint
```

Comment:
This is something that I'm not sure about. Gesture options are dynamically derived based on settings in gesture handlers. And since the focal point option is shared, there are multiple handlers that can populate this option. 
On the other hand, since the focal point option is passed to gesture handlers only through this property, then it can be populated from any of them as all of them should have the same value.

Reply:
Up until now, we'd intentionally avoided having options that applied to more than one gesture on iOS. It's helped to keep the implementation simple and has avoided this problem.

You're right that there's no way for their values to diverge in the current design, so it's probably fine. If we wanted to, we could split this into 4 options, one for each gesture that supports it. The naming would follow the existing pattern:

* <gestureName><Option>
* doubleTapToZoomInFocalPoint
* doubleTouchToZoomOutFocalPoint
* quickZoomFocalPoint
* pinchFocalPoint

This would give greater granularity, but it would diverge from Android.

It would also pave the way towards reorganizing gesture options in the future if we wanted to replace the somewhat monolithic `GestureManager` with a per-gesture object, each with its own `gestureRecgonizer` and `options`.

@pengdev I'd like your input on this since it was a topic we'd discussed last year prior to GA.

Reply:
l would suggest to have an aligned approach here, and in v11, we could add granular control for each gesture.

So we could use the single focal point that control affect all the gestures. On Android, we default to null, which means the focal point will always be the user's touch point.

Reply:
Sounds good, let's proceed with a single option and counting on the handlers staying in sync.